src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Message.vue

```diff
@@ -48,7 +48,7 @@
     </v-sheet>
     <h2>{{ messageStore.currentMessage?.subject }}</h2>
     <div v-for="(message, index) in messageStore.currentThread" :key="index" class="small mt-6">
-      <span v-html="message.from == 'Registry' ? 'From ECE Registry' : 'PortalUser' ? 'You Replied' : ''"></span>
+      <span v-html="message.from == 'Registry' ? 'From ECE Registry' : 'You Replied'"></span>
```

Comment:
I see you modified this check : v-html="message.from == 'Registry' ? 'From ECE Registry' : 'PortalUser' ? 'You Replied' : ''">
 
^^^ is this intentional?? because Initiated from could have these values:
 
 export type InitiatedFrom = "Investigation" | "PortalUser" | "Registry";
 
So if Message is "Initiated From" Investigation it will still show : "You Replied"... No?